tests/apichecks/crds_test.go

```diff
@@ -92,6 +92,8 @@ func TestMissingRefs(t *testing.T) {
 
 // Looks for fields that looks like refs, but are in the status.
 // These fields should not be refs, they should be "external style" links.
+// Persistence of the mutable-but-unreadable SecretKeyRef fields can be exempted
```

Comment:
We want to slowly drop the mutable-but-unreadable field, and not make this an exception. Right now, I suggest we drop the password from the status.observedStaet

Reply:
Done.